internal/write.go

```diff
@@ -0,0 +1,39 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+
+	"github.com/akavel/rsrc/binutil"
+	"github.com/akavel/rsrc/coff"
+)
+
+func Write(coff *coff.Coff, fnameout string) error {
```

Comment:
please add the following comment:

    // TODO(akavel): maybe promote this to coff.Coff.WriteTo(io.Writer) (int64, error)